src/components/generic/fund-card.tsx

```diff
@@ -0,0 +1,134 @@
+import fundCardStyles from "../../styles/generic/fund-card";
+import { FundData } from "../../types";
+
+/**
+ * Interface describing component properties
+ */
+interface Props {
+  fund: FundData;
+}
+
+/**
+ * Component for a fund card
+ *
+ * @param props component properties
+ */
+const FundCard: React.FC<Props> = ({ fund }) => {
+  const styles = fundCardStyles(useTheme());
+  const { color, fundName, priceHistory, lahiTapiola, lastUpdated, riskLevel } = fund;
+
+  /**
+   * Component for price history
+   *
+   * @param label price history label
+   * @param percentage price history percentage
+   * @param lastUpdated last update date on price history
+   */
+  const renderPriceHistory = (label: string, percentage: number) => {
+    return (
+      <View style={ styles.cardColumn }>
+        <Text>
+          { label }
+        </Text>
+        <Text style={ styles.priceHistoryPercentage } >
+          { percentage }
+          %
+        </Text>
+      </View>
+    );
+  };
+  
+  /**
+   * Risk meter
+   */
+  const RiskMeter = () => {
+    return (
+      <View style={{
+        flex: 1,
+        flexDirection: "column",
+        alignItems: "flex-end"
+      }}
+      >
+        <View style={{
+          flex: 1,
+          flexDirection: "row"
+        }}
+        >
+          {Array(riskLevel).fill(
+            <LinearGradient
+              colors={["transparent", "rgba(0,0,0,0.5)"]}
+              style={ styles.riskMeterOn }
+            />
+          )}
+          {Array(7 - Number(riskLevel)).fill(
+            <LinearGradient
+              colors={["transparent", "rgba(0,0,0,0.5)"]}
+              style={ styles.riskMeterOff }
+            />
+          )}
+        </View>
+        <Text>
+          {"Riskitaso "}
+          { riskLevel }
+        </Text>
+      </View>
+    );
+  };
+  
+  /**
+   * Component render
+   */
+  return (
+    <View style={ styles.cardWrapper }>
+      <LinearGradient
+        colors={["transparent", "rgba(0,0,0,0.5)"]}
+        style={{
+          backgroundColor: color,
+          width: "5%",
+          height: "100%"
+        }}
+      />
+      <View style={styles.cardContent}>
+        <View style={ styles.cardRow }>
+          <View style={ styles.cardColumn }>
+            <View style={{ flexDirection: "row" }}>
+              <Text style={ theme.fonts.medium }>
+                { fundName }
+              </Text>
+              {lahiTapiola ? (
+                <Image
+                  // eslint-disable-next-line global-require
+                  source={ require("../../../assets/ltLogo.jpg") }
+                  style={ styles.tinyLogo }
+                />
+              ) : null}
+            </View>
+            <View style={ styles.cardRow }>
+              <Icon name="calendar" size={12} color={ color }/>
+              <Text style={{ paddingLeft: 5 }}>
+                { lastUpdated }
+              </Text>
+            </View>
+          </View>
+          <RiskMeter/>
```

Comment:
Change to render method instead component since it is also defined inside this component